testing/integration/Cargo.toml

```diff
@@ -8,13 +8,13 @@ repository = "https://github.com/filecoin-project/ref-fvm"
 [dependencies]
 fvm = { version = "1.0.0-rc.3", path = "../../fvm", default-features = false }
 fvm_shared = { version = "0.8.0", path = "../../shared" }
-fvm_ipld_hamt = { version = "0.5.1", path = "../../ipld/hamt"}
-fvm_ipld_amt = { version = "0.4.2", path = "../../ipld/amt"}
+fvm_ipld_hamt = { version = "0.5.1", path = "../../ipld/hamt" }
+fvm_ipld_amt = { version = "0.4.2", path = "../../ipld/amt" }
 fvm_ipld_car = { version = "0.4.1", path = "../../ipld/car" }
 fvm_ipld_blockstore = { version = "0.1.1", path = "../../ipld/blockstore" }
 fvm_ipld_encoding = { version = "0.2.2", path = "../../ipld/encoding" }
 
-actors-v7 = { version = "~7.5", package = "fil_builtin_actors_bundle" }
+actors-v8 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors.git", branch = "experimental/fvm-m2" }
```

Comment:
This is tricky. In theory v8 should point to a concrete version now, and we should start a v9 actors on master where we merge our changes to the init actor. However, in reality they will likely be rolled out in v10.

Reply:
https://crates.io/crates/fil_builtin_actors_bundle is a `7.5.1` and master is at `8.0.0-alpha.1`. Should I branch off `experimental/fvm-m2` and bump versions to 9?